packages/core/components/FileList/useFileAccessContextMenu.ts

```diff
@@ -19,6 +21,46 @@ export default function useFileAccessContextMenu(filters?: FileFilter[], onDismi
     const userSelectedApplications = useSelector(interaction.selectors.getUserSelectedApplications);
     const { executionEnvService } = useSelector(interaction.selectors.getPlatformDependentServices);
 
+    const [hasPlateBarCode, setHasPlateBarCode] = React.useState(false);
```

Comment:
Rather than have two variables for similar pieces of information consider converting this to `plateUILink` and consider an `undefined` `plateUILink` to be a sentinel value representing not having a plate barcode for disabling the menu item on line 103

Reply:
[This is the way i initially tried, but couldnt get working](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/47/commits/141ffe9d781553348a903a66a2fb08a6cee09822).

I have to use `useState` so that the view re-renders- but`setPlateLink` never sets `plateLink` when called- am I missing something?

Reply:
I'm not seeing an issue in the commit you linked, that is what I was suggesting and should work